### PR TITLE
Javascript javascript3 week3

### PR DESCRIPTION
**Pull request by ShravyaGurmitkal**

### CODE REVIEW

File: javascript/javascript3/week3/shoppingCart/shoppingCart.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+          this.renderProducts();
+          this.products = productsCopy;
+      } else {
+          productsul.innerHTML = "";
+          this.renderProducts();
+      }
+  }
+
+  getTotal() {
+    // Implement functionality here
```
Comment (simonreddy2001) [inline javascript/javascript3/week3/shoppingCart/shoppingCart.js:68]: "return this.products.reduce((a, b) => a + b.price, 0)"

this also can be used

**Review by simonreddy2001 — APPROVED** [review]
Good job 
Code Works as it should

File: javascript/javascript3/week3/shoppingCart/shoppingCart.js [hunk 2]
```diff
@@ -3,99 +3,99 @@ let productsul = document.createElement('ul');
...
-                currencyPrice.innerHTML = `Price In ${currency}: ${priceinCurrency}`;
-                document.body.appendChild(productName);
-                document.body.appendChild(dkkPrice);
-                document.body.appendChild(currencyPrice);
-            });
-    }
+  convertToCurrency(currency) {
+      // Assuming dkr as default currency
+    const currencyApi = `https://api.currencyapi.com/v3/latest?apikey=8oj1A2di6JFwCRRghtxD1W9zWV4WcA3xVHQN1o7M&currencies=${currency}&base_currency=DKK`;
+    fetch(currencyApi)
```
Comment (RZhyvitskyi) [inline javascript/javascript3/week3/shoppingCart/shoppingCart.js:14]: I prefer to use async / await for a function this long. And I would make an external function that handle all DOM manipulations.

File: javascript/javascript3/week3/shoppingCart/shoppingCart.js [hunk 3]
```diff
@@ -3,99 +3,99 @@ let productsul = document.createElement('ul');
...
+        dkkPrice.innerHTML = `Price In DKK: ${this.price}`;
+        currencyPrice.innerHTML = `Price In ${currency}: ${priceinCurrency}`;
+        document.body.appendChild(productName);
+        document.body.appendChild(dkkPrice);
+        document.body.appendChild(currencyPrice);
+      });
   }
+}
   
 class ShoppingCart {
```
Comment (RZhyvitskyi) [inline javascript/javascript3/week3/shoppingCart/shoppingCart.js:31]: Did you think to put each Class in it's own file? Can be a nice idea :)

**Review by RZhyvitskyi — APPROVED** [review]
Everything is working, good job 👍

File: javascript/javascript3/week3/shoppingCart/shoppingCart.js (hunk 1 above)
Comment (ShravyaGurmitkal) [reply javascript/javascript3/week3/shoppingCart/shoppingCart.js:68]: Yes, I will try it with array.reduce.

**Pull request merged**